src/setupTests.ts

```diff
@@ -8,7 +8,9 @@ const customGlobal: GlobalWithFetchMock = global as GlobalWithFetchMock;
 // eslint-disable-next-line @typescript-eslint/no-require-imports
 customGlobal.fetch = require('jest-fetch-mock');
 customGlobal.fetchMock = customGlobal.fetch;
-
+global.window = {
+    _env_: require('../public/env-config').window
```

Comment:
The `env-config` file in this changeset has the following structure `window._env_`. Shouldn't this line then result in a window object like this:

```js
window: {
  _env_: {
    _env_ : { /* ... */ }
  }
}
```

Reply:
I'm not entirely sure what the `.window` does after the `require`. Doesn't the `env-config` file just mutate the `window` object while actually not exporting anything?

Reply:
That was the way tests were passing. Have to debug those to see how jest makes it's magic.

Reply:
Thought error, require itself was enough to create the object that was used in tests.